tests/routes/search_test.py

```diff
@@ -90,6 +90,42 @@ def test_empty_search_path(self):
             }) \
             .post({}, headers=self.auth)
 
+    def test_sort_no_metadata(self):
+        # Adding metadata tag THINGS to test artifact to ensure it comes first
+        # on DESC search.
+        self.route_tester \
```

Comment:
Can we make this not use the route?  For most route tests i'd like to test a single route at a time unless we have a specific flow that had a bug or something.

In fact, I wouldn't mind if you just added a unit test for something like this instead.  Although that I will leave up to you.